challenge/docker-entrypoint.d/39_start_gui.sh

```diff
@@ -34,6 +34,21 @@ start-stop-daemon --start \
                   </dev/null \
                   >>/tmp/vnc/websockify.log \
                   2>&1
+
```

Comment:
Can we refactor this so that we have a: `38_start_gui_windows.sh`

All of our pre-existing stuff stays here, all of the new windows stuff moves there. Additionally, instead of "2" suffix in all of these paths, lets use "-windows".

Reply:
Additionally, we should do the same "*.exe" check to determine if this should be started.